cypress/e2e/snippets.cy.js

```diff
@@ -67,12 +68,12 @@ describe('Snippets', () => {
 
   it('driven with keyboard', () => {
     // Open and format for insertion point
-    typeCode(`${navigator.platform.match('Mac') ? '{cmd}' : '{ctrl}'}k`);
+    typeCode(`${isMac() ? '{cmd}' : '{ctrl}'}k`);
```

Comment:
I thought a bit more about this and I think these should be split into two test suites. The shortcuts are different for each platform so it makes sense to test them on each platform.

Also, [some people believe](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-in-test.md) (myself included) that there should be no conditional logic in tests.

Reply:
Hmm, I think this is a slightly different scenario than just conditional logic. Agree that we should run both versions, but I wouldn't want the tests getting out of sync because they're separate files.

I might just re-run the cypress step in a macos action, so that we know that both versions are working.

Reply:
Feels like this just moves the ternary up a few levels, and will add a bunch of skipped test alerts to the logs? Like we're still doing conditional logic.